ingestion/src/metadata/ingestion/source/database/unitycatalog/metadata.py

```diff
@@ -487,17 +497,23 @@ def get_columns(self, column_data: List[ColumnInfo]) -> Iterable[Column]:
         """
 
         for column in column_data:
-            if column.type_text.lower().startswith("union"):
-                column.type_text = column.Type.replace(" ", "")
-            if column.type_text.lower() == "struct":
-                column.type_text = "struct<>"
+            parsed_string = {}
+            if column.type_text:
+                if column.type_text.lower().startswith("union"):
+                    column.type_text = column.type_text.replace(" ", "")
+                if (
+                    column.type_text.lower() == "struct"
+                    or column.type_text.lower() == "array"
+                ):
```

Comment:
databricks sends the datatype as `array` or `struct` which leads to errors while parsing as the arrayDataType returns as None